enterprise/models.py

```diff
@@ -764,3 +774,37 @@ def __repr__(self):
         Return uniquely identifying string representation.
         """
         return self.__str__()
+
+
+class EnterpriseIntegratedChannel(TimeStampedModel):
+    """
+    EnterpriseIntegratedChannel is an external system that the Enterprise Customer uses to manage enterprise
+    related data. Each of these requires an edx plugin in order to send data to these systems, this model stores
+    information related to what and where to send data from the enterprise app.
+    """
+    id = models.PositiveIntegerField(primary_key=True)
+    name = models.CharField(max_length=255, blank=False, null=False, help_text=_("Third Party name."))
+    data_type = models.CharField(max_length=100, blank=False, null=False, help_text=_("Data Type"))
+    plugin_endpoint = models.CharField(max_length=255, blank=False, null=False, help_text=_("Internal Plugin endpoint."))
```

Comment:
From the code so far, I can't really tell what `data_type` and `plugin_endpoint` are used for or what sort of values they accept.